src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -26,13 +26,16 @@ const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) =>
       control={control}
       rules={{ required: true }}
       render={({ field: { onChange, value } }) => (
-        <TextField select onChange={onChange} value={value} label="Status">
-          {statuses.map((t) => (
-            <MenuItem key={t} value={t}>
-              {t}
-            </MenuItem>
-          ))}
-        </TextField>
+        <FormControl>
+          <FormLabel>Status</FormLabel>
+          <TextField select onChange={onChange} value={value}>
+            {statuses.map((t) => (
+              <MenuItem key={t} value={t}>
+                {t}
+              </MenuItem>
+            ))}
+          </TextField>
+        </FormControl>
```

Comment:
<img width="783" alt="Screen Shot 2023-02-06 at 12 26 35 AM" src="https://user-images.githubusercontent.com/87815804/216890875-4a75ff6c-5390-462b-9986-f45def365a0e.png">


less important, but when i shrink the screen a little, the formatting gets a little weird